README.md

```diff
@@ -0,0 +1,7 @@
+# pulp_cluster
+
+The `pulp_cluster` repository contains the structure of the cluster subsystem used in PULP chips.
+For more details on the internal architecture, see the README.md in the
+[`pulp` repository](https://github.com/pulp-platform/pulp).
+
+The `doc` folder contains a draw.io schematic, as well as a .pdf of that schematic, outlining the most important components in pulp_cluster, as well as the communication interfaces connecting these. While not a complete overview of all signals, this is meant as a slightly more detailed overview that can assist in development.
```

Comment:
Please make sure the drawing is actually shown in the readme.md . You can have a look at other repos in the group to see how this is done: https://github.com/pulp-platform/bigpulp, https://github.com/pulp-platform/axi/blob/master/doc/axi_xbar.md

Otherwise I fear no one will actually ever look at it.